app/builders/sufia/bootstrap_breadcrumbs_builder.rb

```diff
@@ -9,13 +9,11 @@
 class Sufia::BootstrapBreadcrumbsBuilder < BreadcrumbsOnRails::Breadcrumbs::Builder
   include ActionView::Helpers::OutputSafetyHelper
   def render
-    if @elements.blank?
-      return ""
-    end
+    return "" if @elements.blank?
```

Comment:
[A] I find it curious that Rubocop prefers a one-liner here but not in the `rescue` used in `authorities_controller.rb` [B]